MobileCenterDistribute/MobileCenterDistributeTests/MSDistributeTests.m

```diff
@@ -71,6 +71,10 @@ - (void)setUp {
   [MSKeychainUtil clear];
   self.sut = [MSDistribute new];
   self.settingsMock = [MSMockUserDefaults new];
+  
+  // Mock MSUserDefaults shared method to return settingsMock.
+  id userDefaultsMock = OCMClassMock([MSUserDefaults class]);
```

Comment:
Do you think you can add this stub to MSMockUserDefaults init? That way we'll not face the issue on other classes that's using it.